### PR TITLE
CI: retrieve keys for LLVM repo

### PR DESCRIPTION
**Pull request by orangecms**
Signed-off-by: Daniel Maslowski <info@orangecms.org>

### CODE REVIEW

File: .github/workflows/harvey.yml [hunk 1]
```diff
@@ -25,9 +25,8 @@ jobs:
     - name: install deps
       run: |
         sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-        sudo add-apt-repository 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main' -y
         sudo apt-get update
-        sudo apt-get install -y --force-yes ${{ matrix.cc }} rc qemu go-dep
+        sudo apt-get install -y ${{ matrix.cc }} rc qemu go-dep
```
Comment (gmacd) [inline .github/workflows/harvey.yml:29]: do you need to install  ${{ matrix.cc }}?

Comment (gmacd) [reply .github/workflows/harvey.yml:29]: never mind, will be useful if we need to install newer/other versions

**Review by gmacd — APPROVED** [review]

File: .github/workflows/harvey.yml (hunk 1 above)
Comment (gmacd) [reply .github/workflows/harvey.yml:29]: also won't do anything if the compiler is already installed, so it's fine.

**Review by fhs — APPROVED** [review]
Do we need to add back the `--force-yes` flag now that we're staying on Ubuntu 18.04? The commit message needs to be updated otherwise.

**Comment by orangecms** [comment]
yea I'll make it "will be" ;)

**Pull request merged**